parsers/ESTADISTICO_UT.py

```diff
@@ -0,0 +1,193 @@
+    "Térmico": "oil",
+    "Interconexión": "exchange",
+}
+
+
+def get_data(session: Session) -> Response:
+    """
+    Makes a get request to data url.
+    Parses the response then makes a post request to the same url using
+    parsed parameters from the get request.
+    Returns a requests response object.
+    """
+    pagereq = session.get(DAILY_OPERATION_URL)
+
+    soup = BeautifulSoup(pagereq.content, "html.parser")
+
+    # Find and define parameters needed to send a POST request for the actual data.
+    viewstategenerator = soup.find("input", attrs={"id": "__VIEWSTATEGENERATOR"})[
+        "value"
+    ]
+    viewstate = soup.find("input", attrs={"id": "__VIEWSTATE"})["value"]
+    eventvalidation = soup.find("input", attrs={"id": "__EVENTVALIDATION"})["value"]
+    DXCss = "1_33,1_4,1_9,1_5,15_2,15_4"
+    DXScript = "1_232,1_134,1_225,1_169,1_187,15_1,1_183,1_182,1_140,1_147,1_148,1_142,1_141,1_143,1_144,1_145,1_146,15_0,15_6,15_7"
+    callback_param_init = 'c0:{"Task":"Initialize","DashboardId":"OperacionDiaria","Settings":{"calculateHiddenTotals":false},"RequestMarker":0,"ClientState":{}}'
+
+    postdata = {
+        "__VIEWSTATE": viewstate,
+        "__VIEWSTATEGENERATOR": viewstategenerator,
+        "__EVENTVALIDATION": eventvalidation,
+        "__CALLBACKPARAM": callback_param_init,
+        "__CALLBACKID": "ASPxDashboardViewer1",
+        "DXScript": DXScript,
+        "DXCss": DXCss,
+    }
+
+    datareq = session.post(DAILY_OPERATION_URL, data=postdata)
+
+    return datareq
+
+
+def data_parser(response: Response) -> list[dict]:
+    """
+    Slices the object down to a smaller size then converts to usable json.
+    Loads the data as json then finds the 'result' key.
+    Uses regex to find the start
+    and endpoints of the actual data.
+    Splits the data into datapoints then cleans them up for processing.
+    """
+    double_json = response.text[len("0|/*DX*/(") : -1]
+    double_json = double_json.replace("'", '"')
+    double_json = double_json.replace("\\n", "")
+    double_json = double_json.replace("\\t", "")
+    # Replacing js date objects with isoformat strings.
+    JS_DATE_REGEX = re.compile(
+        r"new Date\((?P<year>\d*),(?P<month>\d*),(?P<day>\d*),(?P<hour>\d*),(?P<minute>\d*),(?P<second>\d*),(?P<ms>\d*)\)"
+    )
+    matches = JS_DATE_REGEX.findall(double_json)
+    if matches:
+        for _match in matches:
+            year, month, day, hour, minute, second, ms = _match
+            dt = datetime(
+                year=int(year),
+                month=int(month) + 1,
+                day=int(day),
+                hour=int(hour),
+                tzinfo=TIMEZONE,
+            )
+            double_json = double_json.replace(
+                f"new Date({year},{month},{day},{hour},{minute},{second},{ms})",
+                f'\\"{dt.isoformat()}\\"',
+            )
+    data = json.loads(double_json)
+    jsresult = data["result"]
+    clean_json = json.loads(jsresult[1:-1])
```

Comment:
By the way, if you keep the old data source there's also a CSV/XLS/XLSX export that might be slightly easier to work with: 
!['Export to Excel' menu on the Generación por tipo de tecnología table](https://github.com/electricitymaps/electricitymaps-contrib/assets/2470175/9961f600-a3cd-4b24-9d17-6785a1af98a5)